### ISSUE TITLE
Release 2024.11.2

### ISSUE THREAD

**Issue by fjetter** [dask/community#405]
|  |  |
| --- | --- |
| Release version | 2024.11.2 |
| Planned release date | 2024-11-13 |
| Status | Unscheduled |
| Release manager | @fjetter |

We again detected a severe regression that was introduced in 2024.11.0 so I suggest to cut an immediate patch release once the PR is merged

## Best effort

_Try to close before the release but will not block the release_

- [ ] ...
- [ ] ...

## Blocker

_Issues that would cause us to block and postpone the release if not fixed_

- [ ] https://github.com/dask/distributed/pull/8933
- [ ] ...

## Comments

...

**Comment by fjetter** [comment]
pypi release is out

**Comment by jrbourbeau** [comment]
Thanks @fjetter -- conda-forge PRs are up too

- [x] https://github.com/conda-forge/dask-core-feedstock/pull/224
- [x] https://github.com/conda-forge/distributed-feedstock/pull/299
- [x] https://github.com/conda-forge/dask-expr-feedstock/pull/86
- [x] https://github.com/conda-forge/dask-feedstock/pull/279
